docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -858,6 +860,19 @@ By default, the users' organization and role is reset on every new login.
 > With Grafana 10, if `oauth_skip_org_role_update_sync` option is set to `false`, users with no mapping will be
 > reset to the default organization role on every login. [See `auto_assign_org_role` option]({{< relref ".#auto_assign_org_role" >}}).
 
+### [auth].<provider> skip_org_role_update_sync
+
+To prevent synchronization of organization roles for a specific OAuth integration, you can set the `skip_org_role_sync` option to `true`. Please note that there is also a separate setting called `oauth_skip_org_role_update_sync` which has a different scope. While `skip_org_role_sync` only applies to the provider integration, `oauth_skip_org_role_update_sync` is a generic setting that affects GitHub, GitLab, AzureAD, and Generic OAuth integrations. However, we are planning to phase out the use of the generic setting in favor of provider-specific settings.
```

Comment:
@Jguer your previews comment was removed infavor of the new doc string instead.

Is it possible to review this text, you mentioned that `independant` was incorrect to explain the differences between these settings.

Reply:
```suggestion
To prevent synchronization of organization roles for a specific OAuth integration, you can set the `skip_org_role_sync` option to `true`. Please note that there is also a separate setting called `oauth_skip_org_role_update_sync` which has a different scope. While `skip_org_role_sync` only applies to the specific OAuth provider, `oauth_skip_org_role_update_sync` is a generic setting that affects all configured OAuth providers. 

 The setting `oauth_skip_org_role_update_sync` is being deprecated in favor of provider-specific settings.
```